api/src/utils/types/attributeTypes.ts

```diff
@@ -51,19 +51,32 @@ export const enum Scope {
 }
 
 export interface UserLoginAttributes {
-    username: string;
-    password: string;
+  username: string;
+  password: string;
 }
 
 export type JsonNetworkType<T> =
   // Types with manual reviving
-  T extends Date ? string :
-  // Carry these through
-  T extends number ? T :
-  T extends boolean ? T :
-  T extends string ? T :
-  T extends JSON ? T :
-  T extends object ? { [K in keyof T]?: JsonNetworkType<T[K]>} :
-  T extends Array<infer E> ? Array<JsonNetworkType<E>> :
-  // Unknown case
-  T;
+  T extends Date
+    ? string
+    : // Carry these through
+    T extends number
+    ? T
+    : T extends boolean
+    ? T
+    : T extends string
+    ? T
+    : T extends JSON
+    ? T
+    : T extends object
+    ? { [K in keyof T]?: JsonNetworkType<T[K]> }
+    : T extends Array<infer E>
+    ? Array<JsonNetworkType<E>>
+    : // Unknown case
+      T;
+
+export type HealthCheckType = {
+  uptime: any;
```

Comment:
S: Refine from `any`, though if converting to `number` would be resolved.

Reply:
Yeah its the wrong type lol

Reply:
Commit: e5b664bca96f28bff5af1d3f474f78dadb02eafb